test/framework/integration_framework/integration_test_framework.cpp

```diff
@@ -220,7 +215,12 @@ namespace integration_framework {
 
   IntegrationTestFramework &IntegrationTestFramework::sendTx(
       const shared_model::proto::Transaction &tx) {
-    sendTx(tx, [](const auto &) {});
+    sendTx(tx, [this](const auto &status) {
```

Comment:
That is pretty arguable fix. Tx error might be considered as "normal case". At least use please `log_->debug`

Reply:
done